calico_node/filesystem/etc/rc.local

```diff
@@ -27,12 +27,22 @@ else
 
     # Run confd twice.  Our confd TOML files are also generated from confd, so
     # running twice ensures our starting configuration is correct.
-    confd -confdir=/etc/calico/confd -onetime -node=${ETCD_SCHEME:=http}://${ETCD_AUTHORITY} \
+    case "$ETCD_ENDPOINTS" in
```

Comment:
There are two cases because that's the simplest way to set ETCD_SCHEME when endpoints is in use.  I used a case, not something shorter because it's not specific to any shell, and is quite easy to understand even if you aren't a *sh expert.

That leaves the question of why do we need ETCD_SCHEME.  Well, currently we pass it to confd separately (-scheme).

I've looked at the confd code, and -scheme is used to get the nodes from DNS.  We don't do this, so I think we can just retire -scheme.